documentation/news.md

```diff
@@ -0,0 +1,47 @@
+# News & updates
```

Comment:
When do we need to update this file?

Reply:
New codelabs, new pages to documentation, announcements, or bigger updates to the project that would either not be covered by changelog (e.g. codelabs is external), or noteworthy enough to not be lost among a 100 other changes.

Reply:
so for every PR we need to potentially keep this in mind?